hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/ha/HadoopRpcOMFailoverProxyProvider.java

```diff
@@ -75,8 +74,7 @@ public HadoopRpcOMFailoverProxyProvider(ConfigurationSource configuration,
                                  UserGroupInformation ugi,
                                  String omServiceId,
                                  Class<T> protocol) throws IOException {
-    super(configuration, omServiceId, protocol);
-    this.ugi = ugi;
+    super(configuration, ugi, omServiceId, protocol);
     this.omVersion = RPC.getProtocolVersion(protocol);
```

Comment:
`omVersion` is no longer used.

Reply:
Addressed in [c52deae](https://github.com/apache/ozone/pull/7268/commits/c52deaec7d51ec9adea755d75d83e82bcdd74e3d)